lib/vector-buffers/src/test/common/messages.rs

```diff
@@ -116,7 +116,11 @@ impl FixedEncodable for SizedRecord {
         if buffer.remaining_mut() < self.0 as usize + 4 {
             return Err(io::Error::new(
                 io::ErrorKind::Other,
-                "not enough capacity to encode record",
+                format!(
+                    "not enough capacity to encode record: need {}, only have {}",
+                    self.0 + 4,
```

Comment:
4 for the `u32`? Might be worth pulling the `size_of` for self.0 instead.

Reply:
Yeah, totally fair point. I'll make this a bit more robust.